engine/src/main/resources/templates/AnsibleServiceOnAws/notifyCustomerProvisionSuccessEmailTitleV2.txt

```diff
@@ -0,0 +1 @@
+Instant notification - Notify customer environment is ready - Ansible Automation Platform Service on AWS - Ansible Automation Platform
```

Comment:
```suggestion
Instant notification - Environment is ready - Ansible Automation Platform Service on AWS - Ansible Automation Platform
```

Reply:
It supposed to be a fix format with "Instant notification - EventTypeDisplayName - ApplicationDisplaName - BundleName", I prefer let it as it is and update it later regarding what Ansible team will decide if it's ok for you?

Reply:
Ah yeah, that's fine then!